docs/comments_endpoints.md

```diff
@@ -72,11 +128,52 @@ Response
 ```ruby
 Status: 201
 {
-  message: "Successfully added"
+  id: 4,
+  text: 'This is such a scam',
+  date_created: 'Wed, 25TH Nov, 2017 12:00PM',
+  updated: true,
+  score: 9,
+  user: {
+    id: 3,
+    name: 'Bayo Owoade'
```

Comment:
It's possible; but I'd suggest we leave that till we are done with the switch to `jbuilder` and then we could even use a standard API documentation tool.